website/source/docs/agent/configuration/consul.html.md

```diff
@@ -41,6 +41,11 @@ cluster with zero configuration. To put it another way: if you have a Consul
 agent running on the same host as the Nomad agent with the default
 configuration, Nomad will automatically connect and configure with Consul.
 
+If you connect multiple nodes to a single consul server, by using
```

Comment:
How about:

An important requirement is that each Nomad agent talks to a unique Consul agent. Nomad agent's should be configured to talk to Consul agents and not Consul servers. If you are observing flapping services, you may have have multiple Nomad agent's talking to the same Consul agent. As such avoid configuring Nomad to talk to Consul via DNS such as `consul.service.consul`